flow-api/src/main/java/com/sequenceiq/flow/api/model/FlowCheckResponse.java

```diff
@@ -15,6 +15,12 @@ public class FlowCheckResponse {
 
     private Long endTime;
 
+    private String currentState;
+
+    private String nextEvent;
+
+    private String flowType;
```

Comment:
please add toString implementation

Reply:
Added a toString implementation for FlowCheckResponse. Let me know if it looks good!